packages/app/src/systems/Error/pages/ReportErrors/ReportErrors.tsx

```diff
@@ -0,0 +1,138 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Dialog,
+  IconButton,
+  Icon,
+  Stack,
+  FuelLogo,
+  Heading,
+  Button,
+  Box,
+  Text,
+} from '@fuel-ui/react';
+
+import { useReportError } from '../../hooks';
+
+import { WALLET_WIDTH, WALLET_HEIGHT } from '~/config';
+import { coreStyles } from '~/systems/Core/styles';
+
+export function ReportErrors() {
+  const {
+    handlers,
+    isLoadingDontSend,
+    isLoadingSendAlways,
+    isLoadingSendOnce,
+  } = useReportError();
+
+  const isLoading =
+    isLoadingDontSend || isLoadingSendAlways || isLoadingSendOnce;
+
+  return (
+    <>
+      <Dialog.Heading>
+        Report Errors
+        <IconButton
+          data-action="closed"
+          variant="link"
+          icon={<Icon icon="X" color="gray8" />}
```

Comment:
instead of gray should we consider using intentsBase?

Reply:
@matt-user We don't have `intent` in `fuel-ui v0.12` that this PR is using. 
Fuel UI v0.14 has breaking changes, so I can't do the upgrade on this PR. There'll be another task for that.